### PR TITLE
Fixing bug for issue#26

### PR DESCRIPTION
**Pull request by grandmaJ**

### CODE REVIEW

File: public/scripts/main.js [hunk 1]
```diff
@@ -1,184 +1,200 @@
...
+   }
 }
 
 createWordleBoard()
 // Choosing a random word from the list
 const len = wordList.length
 const chosenWord = wordList[Math.floor(Math.random() * len)]
-
-
+console.log(chosenWord)
```
Comment (Zakhna) [inline public/scripts/main.js:28]: This needs to be removed.

**Comment by Zakhna** [comment]
Looks good, the bug seems to be resolved. I shall just take out unnecessary lines of code. We should also implement tests for this functionality in the next sprint.

**Review by Zakhna — APPROVED** [review]
Everything looks good. Well done!

**Comment by grandmaJ** [comment]
> Looks good, the bug seems to be resolved. I shall just take out unnecessary lines of code. We should also implement tests for this functionality in the next sprint.

Thanks!! I must have forgotten about those commented out lines of code. Great that you picked up on it!

**Pull request merged**